test/data/router-methods-helper.js

```diff
@@ -0,0 +1,9 @@
+self.getMethodToTest = () => {
+  const methodRegex = /\?method=(\w*)/;
```

Comment:
Could you switch this to `/[?&]method=(\w+)/` to ensure that it would work if `method` isn't the first parameter, and reinforce the fact that the value can't be empty (`+` vs. `*`).